bedrock/firefox/templates/firefox/desktop/index-b.html

```diff
@@ -0,0 +1,31 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/desktop/index.html" %}
+
+{% block email_form %}
+  <aside id="subscribe-download-wrapper" class="has-variant ga-section" data-ga-label="Get Firefox news">
+    <div class="content container">
+      <section class="variant">
+        <header>
+        {% if LANG.startswith('en-') %}
+          <h3>{{ _('Keep up with all things Firefox.') }}</h3>
+        {% else %}
+          <h3>{{ _('Get Firefox news') }}</h3>
+        {% endif %}
+        </header>
+        {{ email_newsletter_form(title=' ') }}
+      </section>
+
+      <section class="original">
+        <div id="subscribe-wrapper">
+          {{ email_newsletter_form(title=_('Get Firefox news')) }}
```

Comment:
Hmm you're right, of course. I knew repeating it was sloppy but hadn't thought about how it would break the ajax. I'll rework it to avoid the repetition.